docs/en/get_started/ascend/get_started.md

```diff
@@ -1,85 +1,46 @@
-# Get Started with Huawei Ascend (Atlas 800T A2 & Atlas 300I Duo)
+# Get Started with Huawei Ascend (Atlas 800T A3 & Atlas 800T A2 & Atlas 300I Duo)
```

Comment:
How about entitle `Get Started with Huawei Ascend` and leave the device in the text body?